urld025f21ac3_mvattipulusu_Innovation2013_tgz-pJ8-com_usps_data_ManifestFileTestJ8/src/com/usps/evs/vo/SampleHeader.java

```diff
@@ -0,0 +1,377 @@
+    String userId;
+    String zeroVolumeFlag;
+    int recordCount;
+    String corpDuns;
+    String evsVersion;
+    String companyDatVersion;
+    String returnType;
+    int week;
+    String fiscalMonth;
+    String fiscalYear;
+    int systemType;
+    String filename;
+    String filetype;
+    boolean evsLite;
+	String siteID;
+	String costCenter;
+	String headerString;
+	private String tpbFl = "N"; // By-for relationship
+	
+    /** Creates a new instance of SampleHeaderVO */
+    public SampleHeader()
+    {
+    }
+    
+	public String toString(){
+		ToStringBuilder.setDefaultStyle(ToStringStyle.MULTI_LINE_STYLE);
+		return ToStringBuilder.reflectionToString(this).toString();
+    	
+	}
+	
+    /**
+     * Getter for property zipCode.
+     * @return Value of property zipCode.
+     */
+    public java.lang.String getZipCode() {
+        return zipCode;
+    }
+    
+    /**
+     * Setter for property zipCode.
+     * @param zipCode New value of property zipCode.
+     */
+    public void setZipCode(java.lang.String zipCode) {
+        this.zipCode = zipCode;
+    }
+    
+    /**
+     * Getter for property facilityType.
+     * @return Value of property facilityType.
+     */
+    public java.lang.String getFacilityType() {
+        return facilityType;
+    }
+    
+    /**
+     * Setter for property facilityType.
+     * @param facilityType New value of property facilityType.
+     */
+    public void setFacilityType(java.lang.String facilityType) {
+        this.facilityType = facilityType;
+    }
+    
+    /**
+     * Getter for property userId.
+     * @return Value of property userId.
+     */
+    public java.lang.String getUserId() {
+        return userId;
+    }
+    
+    /**
+     * Setter for property userId.
+     * @param userId New value of property userId.
+     */
+    public void setUserId(java.lang.String userId) {
+        this.userId = userId;
+    }
+    
+    /**
+     * Getter for property zeroVolumeFlag.
+     * @return Value of property zeroVolumeFlag.
+     */
+    public java.lang.String getZeroVolumeFlag() {
+        return zeroVolumeFlag;
+    }
+    
+    /**
+     * Setter for property zeroVolumeFlag.
+     * @param zeroVolumeFlag New value of property zeroVolumeFlag.
+     */
+    public void setZeroVolumeFlag(java.lang.String zeroVolumeFlag) {
+        this.zeroVolumeFlag = zeroVolumeFlag;
+    }
+    
+    /**
+     * Getter for property recordCount.
+     * @return Value of property recordCount.
+     */
+    public int getRecordCount() {
+        return recordCount;
+    }
+    
+    /**
+     * Setter for property recordCount.
+     * @param recordCount New value of property recordCount.
+     */
+    public void setRecordCount(int recordCount) {
+        this.recordCount = recordCount;
+    }
+    
+    /**
+     * Getter for property evsVersion.
+     * @return Value of property evsVersion.
+     */
+    public java.lang.String getEvsVersion() {
+        if(this.evsVersion != null)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It seems unusual to set the value of a variable only when it is already set. You might want to switch the null check of != on line 143 to == check before setting the field value on line 144 or verify if the check is needed at all.